src/common/actions/repositories.js

```diff
@@ -11,10 +11,13 @@ export const REPOSITORIES_SUCCESS = 'REPOSITORIES_SUCCESS';
 export const REPOSITORIES_FAILURE = 'REPOSITORIES_FAILURE';
 
 
-export function fetchRepositoriesSuccess(repos) {
+export function fetchRepositoriesSuccess(result) {
   return {
     type: REPOSITORIES_SUCCESS,
-    payload: repos
+    payload: {
+      ...result.payload,
+      pageLinks: result.pageLinks
```

Comment:
Added `pageLinks` to action payload, because this is where the reducer expects them to be.